### ISSUE TITLE
Will not begin downloading.

### ISSUE THREAD

**Issue by RPMYT** [goatcorp/FFXIVQuickLauncher#1425]
### Update disclaimer

- [X] Yes, I have checked and my issue is not related to the game updating and plugins not working correctly.

### What did you do?

Trying to download the game.
See image:
![image](https://github.com/goatcorp/FFXIVQuickLauncher/assets/45874270/1e1c4657-c93f-4b86-9f31-dc27f95b88af)

### Platform

Linux (native)

### Wine/Proton runner version

Latest

**Comment by NotNite** [comment]
This technically belongs on the XIVLauncher.Core repository, but this isn't a bug; this is just your internet. Have you tried the suggested troubleshooting steps?

**Comment by RPMYT** [comment]
What troubleshooting steps? Where?

**Comment by NotNite** [comment]
The one on the error message - attempting use of a VPN (or another network).

**Comment by RPMYT** [comment]
Is it possible to skip verification and download anyway?
At least then I can see if it actually doesn't download or if it just fails a checksum somewhere...

**Comment by reiichi001** [comment]
Because your device is having trouble reaching sites/services hosted by Square Enix. THe official launcher is really not much more than a wrapper for Internet Explorer. (IE4, then MSHTML).

[ProtonVPN](https://protonvpn.com/) or [Cloudflare WARP](https://1.1.1.1/) are both free to use to try out. Or if your ISP has broken IPv6 support, you can try telling your OS to prioritize IPv4.  https://weblog.lkiesow.de/20220311-make-linux-prefer-ipv4.html

The hash check failure is from the patch list SE provides. If that fails, it means the patch downloaded is corrupted. There no reason to apply a known corrupted patch. That's just asking for it to get worse.

**Comment by RPMYT** [comment]
ProtonVPN requires the use of NetworkManager; and I don't think WARP has JP servers.
I highly doubt it will work.

**Comment by RPMYT** [comment]
In fact neither WARP nor the Wireguard compatibility tool work;
WGCF states `Unable to access interface: Protocol not supported` and cloudflare WARP simply doesn't do anything even when I explicitly `warp-cli connect` - `curl https://www.cloudflare.com/cdn-cgi/trace/` reports `warp=off`.

**Comment by RPMYT** [comment]
Nor does the [unofficial ProtonVPN CLI.](https://github.com/Rafficer/linux-cli-community)

**Comment by reiichi001** [comment]
> ProtonVPN requires the use of NetworkManager; and I don't think WARP has JP servers. I highly doubt it will work.

Only the login server is in Japan, which is why we mention that in common troubleshooting error messages. Patches are handled by Akamai's CDN globally. Anything to get you onto a different CDN is honestly going to be good enough. 

You're free to use any other VPN service. Those are just two we usually pitch because they're free and this isn't about trying to cheat regional content distribution or encrypting network traffic or the latest buzzwords VPN companies are sponsoring youtube content creators to read out.

This sounds like a larger issue with your current distro's setup or your ISP, which aren't really within our control or ability to troubleshoot.

**Comment by RPMYT** [comment]
> You're free to use any other VPN service.

I don't think this is an issue with my network, then; because even tunnelling all connections through `sshuttle` it still does not work - I even checked my IP changed.
